package.json

```diff
@@ -11,10 +11,8 @@
     "snyk": "snyk test",
     "clean": "rimraf ./dist && mkdir -p ./dist ./dist/js ./dist/css ./dist/fonts ./dist/img",
     "start": "./server/server.js",
-    "js": "browserify -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] ./src/js/index.js -d | ./node_modules/exorcist/bin/exorcist.js ./dist/js/app.min.js.map > ./dist/js/app.min.js",
-    "prod:js": "browserify -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] ./src/js/index.js | uglifyjs > ./dist/js/app.min.js",
+    "js": "browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js > ./dist/js/app.min.js",
     "less": "lessc --source-map-map-inline ./src/less/app.less ./dist/css/app.min.css && cp node_modules/cf-icons/src/fonts/* ./dist/fonts",
```

Comment:
For the less source map I think we want an external file too. Otherwise the file size will be a bit much.
So instead of `--source-map-map-inline` we can use `--source-map=./dist/css/app.min.css.map`.

Also add the `--clean-css` flag to output the minified file.